api-contracts/api-contract.yml

```diff
@@ -1,36 +1,144 @@
-swagger: '2.0'
+openapi: 3.0.0
 info:
-  version: 1
+  version: "1"
   title: Harald
   description: The one notification
-consumes:
-  - application/json
-produces:
-  - application/json
-basePath: /api/v1/
 paths:
   /messages:
     post:
       summary: Queue a message for dispatching
-      parameters:
-        - in: body
-          required: true
-          schema:
-            $ref: "#/definitions/message"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/message"
+        required: true
       responses:
-        202:
+        "202":
           description: Accepted - message is queued for dispatching.
-        400:
+        "400":
           description: Bad request - required input is missing.
-        422:
+        "422":
           description: Unprocessable entity - capability ID can't be processed.
-        503:
+        "503":
           description: Service unavailable - error trying to send the message.
-definitions:
-  message:
-    type: object
-    properties:
-      capabilityId:
-        type: string
-      message:
-        type: string
\ No newline at end of file
+  /connections:
+    get:
+      summary: A queryable collection of connections between publishers and
+        communication channels
+      parameters:
+        - in: query
+          name: senderType

```

Comment:
I actually think you should call this publishType to make it more inline with the summary.

Reply:
DO we think we have a publisher that publishes on a channel or do we have a sender that sends messages on a channel?

Reply:
Client?